lib/accelerometer.js

```diff
@@ -167,7 +167,7 @@ var Controllers = {
   MMA7361: {
     DEFAULTS: {
       value: {
-        zeroV: [336, 372, 287],
+        zeroV: [330, 350, 290],
```

Comment:
> Sure. I did this because the calibration is never going to be perfect for all users.

True, which is why Accelerometer allows for user specified zeroV:

```js
var accelerometer = new five.Accelerometer({
  controller: "MMA7361",
  pins: ["A0", "A1", "A2"],
  zeroV: [zeroVx, zeroVy, zeroVz],
});
```


> If we're going to pick values based on the first person's calibration (I'm assuming that's where these numbers came from), 

TBH, I don't know how the default zeroV numbers were derived, but they were correct for both @BrianGenisio (who authored the controller) and myself. 


> An MMA7361 in an Esplora shouldn't really have different values than an MMA7361 in other settings. 

That's not strictly true. Any number of differences in the circuit that the chip is attached to could cause slight variations.

> FWIW, the values that ship in Esplora.cpp are [320,330,310].


Ah, in that case: 

```js
Controllers["ESPLORA"] = {
  DEFAULTS: {
    value: {
      zeroV: [320, 330, 310], 
      sensitivity: 170
    }
  }, 
  initialize: {
    value: function(opts, dataHandler) {
      opts.pins = [5, 11, 6];
      analogInitialize.call(this, opts, dataHandler);
    }
  },
  toGravity: Controllers.MMA7361.toGravity
};
```